examples/amd/example_amd_flash_attn_bwd.py

```
@@ -1,363 +1,594 @@
+    post_kernel = flashattn_bwd_postprocess(batch, heads, seq_len, dim)
+    dQ_tl = post_kernel(dQ_accum)
+
+    q_ref = q.clone().detach().requires_grad_()
+    k_ref = k.clone().detach().requires_grad_()
+    v_ref = v.clone().detach().requires_grad_()
+
+    o_ref, _ = ref_program(q_ref, k_ref, v_ref, is_causal, groups)
+    o_ref.backward(dO)
+
+    print("Verifying backward pass correctness...")
+    dq_close, dq_max_diff, dq_mean_diff = debug_tensor_comparison(
```

Ruff (0.13.1)

Unpacked variable `dq_mean_diff` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)